airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/clickhouse/test_simple_streams/first_output/airbyte_incremental/scd/test_normalization/dedup_cdc_excluded_scd.sql

```diff
@@ -65,7 +66,7 @@ dedup_data as (
         -- additionally, we generate a unique key for the scd table
         row_number() over (
             partition by _airbyte_unique_key, _airbyte_start_at, _airbyte_emitted_at, accurateCastOrNull(_ab_cdc_deleted_at, 'String'), accurateCastOrNull(_ab_cdc_updated_at, 'String')
-            order by _airbyte_ab_id
+            order by _airbyte_active_row desc, _airbyte_ab_id
```

Comment:
is it expected that we're now sorting on `active_row` in additional to `ab_id`?

Reply:
Adding this prevents cases having multiple `_airbyte_ab_id` where there is some rows with 0 (not active) and making sure to select the active row.